Documentation/Functions/Select.rst

```diff
@@ -117,12 +117,20 @@ uidInList
 
    **Note:** :typoscript:`this` is a *special keyword* and replaced with the id of the
    *current record*.
+   
+   .. important:: 
+      :ref:`pidInList` has to be set or there will be no results. Setting
```

Comment:
Maybe we should emphasize the default `this` for `pidInList` instead. There is actually no relation between these two options, both have their own unrelated behavior.

Reply:
Ok, well I never had this case before. Did you verify? Usually records are stored in sys_folders while the DataProcessor is run on pages. So I see almost no case where setting uidInList without pidInList with results

Reply:
Ok, so I added the default value to pidInList (as we didn't have it documented yet) and changed the important text accordingly